### ISSUE TITLE
Update dataset metadata in Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell-curation#36]
The datasets in [Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney](https://cellxgene.cziscience.com/collections/9b02383a-9358-4f0f-9795-a891ec523bcc) should be corrected to:

- [ ] Use 'human adult stage' rather than 'adult' for the `development_stage label` as indicated by HsapDv:0000087

**Comment by pablo-gar** [comment]
Adding @MaximilianLombardo  as he curated that dataset

**Comment by MaximilianLombardo** [comment]
Unassigning myself because this dataet was curated externally

**Comment by jahilton** [comment]
During migration, we will remove the `development_stage`, and the Portal will add the correct label associated with `HsapDv:0000087`.
Is this ticket still necessary?

**Comment by jahilton** [comment]
This was updated during the migration resubmission
